<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageStichedViewer</class>
 <widget class="QWidget" name="stichedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>302</height>
   </rect>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Cloud</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPointCloud">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="exportCSVPlan">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>155</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Plan .csv</string>
   </property>
  </widget>
  <widget class="QToolButton" name="importCSVPlan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Plan .csv</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxPlan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>m</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonFront">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Front View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewYpos.png</normaloff>:/CC/images/ccViewYpos.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="chooseImage">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccPointPicking.png</normaloff>:/CC/images/ccPointPicking.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Node Pose</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Plan size</string>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonBack">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Back View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewYneg.png</normaloff>:/CC/images/ccViewYneg.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonRight">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Right View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewXneg.png</normaloff>:/CC/images/ccViewXneg.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonLeft">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Left View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewXpos.png</normaloff>:/CC/images/ccViewXpos.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonFloor">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Floor View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewZneg.png</normaloff>:/CC/images/ccViewZneg.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="poseCenterButtonRoof">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>23</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Roof View Cam</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/CC/images/ccViewZpos.png</normaloff>:/CC/images/ccViewZpos.png</iconset>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxOff">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>m</string>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Far</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxFar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="suffix">
    <string>m</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="value">
    <number>90</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxNodeSpeed">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
